FTH/urls.py

```diff
@@ -14,8 +14,11 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path,include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('accounts/',include('accounts.urls', namespace='accounts')),
+    path('',include('home.urls', namespace='home')),
+    path('login/github/', include('social_django.urls', namespace='social')),
```

Comment:
please put the last “login/github/" URL pattern into the accounts.urls. That belongs to the app "accounts".